risc-v-demo/sev-kit-reference-design/script_support/additional_configurations/smarthls/hls_pipeline/compile_and_copy.bat

```diff
@@ -0,0 +1,80 @@
+@echo off
+setlocal EnableDelayedExpansion
+
+REM E.g.:
+REM   cd sev-kit-reference-design/script_support/additional_configurations/smarthls/hls_pipeline
+REM   datapath=fpga arch=riscv_64 compile_and_copy.sh
+REM   datapath=cpu  arch=riscv_64 compile_and_copy.sh 
```

Comment:
Shouldn't this use the Windows calling convention instead of Linux?

"CALL compile_and_copy.bat fpga riscv_64"
"CALL compile_and_copy.bat cpu  riscv_64"

Reply:
Resolved: https://github.com/MicrochipTech/fpga-hls-examples/pull/56/commits/efc36140eb5a18019730786e6ce7a9a4466c61b2